### PR TITLE
Google+ button fixed

### LINKED ISSUE #5727: Disconnect from Google+ is totally broken
screen shot 2015-09-23 at 19 39 36

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #5727
Now users with Google+ login will see its account options correctly.

screen shot 2015-09-28 at 17 51 44

CR @xavijam

### CODE REVIEW

**Comment by xavijam** [comment]
If you are connected you can't see the login button, only the disconnected :S.

File: app/assets/stylesheets/common/account_forms.css.scss [hunk 1]
```diff
@@ -670,3 +670,7 @@ $sLabel-width: 140px;
     text-decoration: underline;
   }
 }
+
+.FormAccount-GooglePlus-iframe {
+  display: none
```
Comment (xavijam) [inline app/assets/stylesheets/common/account_forms.css.scss]: That semicolon!

**Comment by MariaCheca** [comment]
Tested in staging, all working.

screen shot 2015-09-28 at 17 51 44

**Comment by xavijam** [comment]
Add the semicolon, wait for tests, and ready to go :es:.

**Comment by xavijam** [comment]
And change the description image! :D

**Comment by MariaCheca** [comment]
:sparkles:

**Pull request merged**